notify/sns/sns_test.go

```diff
@@ -14,11 +14,19 @@
 package sns
 
 import (
+	"context"
+	"github.com/go-kit/log"
 	"testing"
+	"unicode/utf8"
 
+	"github.com/prometheus/alertmanager/config"
+	"github.com/prometheus/alertmanager/notify/test"
+	commoncfg "github.com/prometheus/common/config"
 	"github.com/stretchr/testify/require"
 )
 
+var logger = log.NewNopLogger()
```

Comment:
can we avoid this as a top level variable?

Reply:
done